src/schema/v2/sale/display.ts

```diff
@@ -100,3 +100,73 @@ export async function displayTimelyAt({ sale, meBiddersLoader }) {
     }
   }
 }
+
+const defaultStartAt = !__TEST__
+  ? new Date().toISOString()
+  : "2020-08-20T02:50:09+00:00"
+
+/**
+ * Get sale ending urgency tag
+ * @example
+ * 2 days left, 12 hours left
+ * Auction closed
+ */
+
+export const displayUrgencyTag = ({
+  endAt,
+  auctionState,
+  startAt = defaultStartAt,
```

Comment:
what does `startAt` represent the start of? Seems like it's used as 'the current time'. In which case I think you should not use this `defaultStartAt` property, which doesn't ever change, and instead have a `getCurrentTime()` function so that it stays up to date for every request.

Reply:
My intention was to make the function return the urgency tag between any random dates, but  you're right, it makes more sense to call it `currentTime` and to switch `defaultStartAt` to a function (`getCurrentTime()`) that is up to date for every request